pkg/sdn/plugin/controller.go

```diff
@@ -468,24 +481,30 @@ func (plugin *OsdnNode) AddServiceRules(service *kapi.Service, netID uint32) {
 func (plugin *OsdnNode) DeleteServiceRules(service *kapi.Service) {
 	glog.V(5).Infof("DeleteServiceRules for %v", service)
 
+	// Remove blanket rule allowing subsequent IP fragments; since we don't
+	// have the service VNID here we have to ignore errors since this will
+	// fail when removing services in the admin VNID
 	otx := plugin.ovs.NewTransaction()
+	otx.DeleteFlows(generateBaseServiceFragRule(service.Spec.ClusterIP))
+	otx.EndTransaction()
+
+	otx = plugin.ovs.NewTransaction()
 	for _, port := range service.Spec.Ports {
-		otx.DeleteFlows(generateDeleteServiceRule(service.Spec.ClusterIP, port.Protocol, int(port.Port)))
+		otx.DeleteFlows(generateBaseServiceRule(service.Spec.ClusterIP, port.Protocol, int(port.Port)))
 		if err := otx.EndTransaction(); err != nil {
 			glog.Errorf("Error deleting OVS flows for service %v: %v", service, err)
 		}
 	}
 }
 
-func generateBaseServiceRule(IP string, protocol kapi.Protocol, port int) string {
-	return fmt.Sprintf("table=60, %s, nw_dst=%s, tp_dst=%d", strings.ToLower(string(protocol)), IP, port)
+func generateBaseServiceFragRule(IP string) string {
+	return fmt.Sprintf("table=60, ip, nw_dst=%s, ip_frag=later", IP)
 }
 
-func generateAddServiceRule(netID uint32, IP string, protocol kapi.Protocol, port int) string {
-	baseRule := generateBaseServiceRule(IP, protocol, port)
-	return fmt.Sprintf("%s, priority=100, actions=load:%d->NXM_NX_REG1[], load:2->NXM_NX_REG2[], goto_table:80", baseRule, netID)
+func generateBaseServiceRule(IP string, protocol kapi.Protocol, port int) string {
+	return fmt.Sprintf("table=60, %s, nw_dst=%s, tp_dst=%d", strings.ToLower(string(protocol)), IP, port)
```

Comment:
also, while we're changing things, can you fix this to use `udp_dst`/`tcp_dst` since `tp_dst` is apparently deprecated?